content/en/blog/_posts/2018-05-23-Airflow-Kubernetes-Operator.md

```diff
@@ -0,0 +1,139 @@
+---
+layout: blog
+title:  The Kubernetes "Whatever-your-heart-desires" Operator
+date:   Wednesday, May 23rd 2018
+---
+
+## Introduction
+
+Today, we are excited to announce the Kubernetes Airflow Operator; a new mechanism for launching containers using the Kubernetes cloud deployment framework. 
```

Comment:
We should disambiguate the k8s and airflow operator concepts here - since most previous blog posts would have dealt with the k8s notion of operator which will throw people off.

Reply:
Any recommendations on how we should draw the distinction? Perhaps pointing to the airflow documentation?

Reply:
That's a good idea. We should point to airflow documentation and then immediately explain what an airflow operator is in 1-2 lines.